src/upb/gen_build_yaml.py

```diff
@@ -29,15 +29,15 @@
         'build': 'all',
         'language': 'c',
         'src': [
+            "third_party/upb/upb/decode_fast.c",
             "third_party/upb/upb/decode.c",
+            "third_party/upb/upb/def.c",
             "third_party/upb/upb/encode.c",
```

Comment:
optional: it helps if we can add `third_party/upb/upb/json_encode.*` and `third_party/upb/upb/json_decode.*`. So, I don't need to touch the upb generation process in another PR. If the JSON codec can make into this upgrade, we should also update `bazel/grpc_deps.bzl` with:

```python
native.bind(
    name = "upb_json_lib",
    actual = "@upb//:json",
)
```

Reply:
Done!